src/Components/SuperAdmins/Form/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import styles from './super-admins.module.css';
+
+const Form = (props) => {
+  if (!props.showForm) {
+    return null;
+  }
+
+  const isPut = props.method === 'PUT';
+
+  const [item, setItem] = useState({
+    firstName: '',
+    email: '',
+    password: ''
+  });
+
+  if (isPut) {
```

Comment:
Instead of having a useEffect inside an if condition, it would be better to move the condition inside the useEffect and add a dependency for isPut. This way, the useEffect will only be executed when isPut changes value and this improves the readability of the code.
```suggestion
useEffect(() => {
  if (isPut) {
    const item = props.data.filter((item) => item._id === props.itemId);
    setItem({
      firstName: item[0].firstName,
      email: item[0].email,
      password: item[0].password
    });
  }
}, [isPut, props.data, props.itemId]);
```